### PR TITLE
Add mediaImage Parameter

### LINKED ISSUE #158: [SDL 0223] Add Currently Playing Media Image to MediaServiceData
Proposal: [Add Currently Playing Media Image to MediaServiceData](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0223-media-service-image.md)

> This proposal adds an image for the currently playing media item to the media app service.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/706

### PR DESCRIPTION
**Pull request by Jack-Byrne**
Note: Targeting master as the base because there are no 5.2 branches yet

Fixes #158

### CODE REVIEW

**Review by jacobkeeler — APPROVED** [review]

**Comment by joeygrover** [comment]
FYI this should target a new release branch not master

File: MOBILE_API.xml [hunk 1]
```diff
@@ -3830,6 +3830,13 @@
                 Audiobook: The total number of sections in the playback queue (e.g. the number of chapters in the book)
             </description>
         </param>
+        <param name="mediaImage" type="Image" mandatory="false" since="5.2">
```
Comment (theresalech) [inline MOBILE_API.xml]:
```suggestion
        <param name="mediaImage" type="Image" mandatory="false" since="6.0">
```

**Review by theresalech — CHANGES_REQUESTED** [review]

File: MOBILE_API.xml [hunk 2]
```diff
@@ -3830,6 +3830,13 @@
                 Audiobook: The total number of sections in the playback queue (e.g. the number of chapters in the book)
             </description>
         </param>
```
Comment (theresalech) [inline MOBILE_API.xml:3832]: Does there need to be a line break here, like there is between other params?

**Review by theresalech — CHANGES_REQUESTED** [review]
Should the README be updated with this new param as well?

File: MOBILE_API.xml (hunk 2 above)
Comment (Jack-Byrne) [reply MOBILE_API.xml:3832]: Updated

**Review by theresalech — APPROVED** [review]

**Review by joeygrover — APPROVED** [review]

**Pull request merged**